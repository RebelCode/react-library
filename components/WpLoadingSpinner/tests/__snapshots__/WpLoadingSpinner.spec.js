import React from "react";
import { render, cleanup } from "@testing-library/react";
import { WpLoadingSpinner } from "../../index";

afterEach(cleanup);

describe("Skeleton", () => {
  const Comp = (
    <WpLoadingSpinner />
  );

  it("should take a snapshot", () => {
    const { asFragment } = render(Comp);

    expect(asFragment(Comp)).toMatchSnapshot();
  });

  it("renders correctly", () => {
    const { queryByLabelText } = render(Comp);

    expect(queryByLabelText("WpLoadingSpinner")).toBeTruthy();
  });
});
