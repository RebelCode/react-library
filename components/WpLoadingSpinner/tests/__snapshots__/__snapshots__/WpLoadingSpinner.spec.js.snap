// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skeleton should take a snapshot 1`] = `
<DocumentFragment>
  <span
    aria-label="WpLoadingSpinner"
    class="loading-spinner"
    style="width: 24px; height: 24px; box-shadow: 6px 0 0 9px #999 inset;"
  />
</DocumentFragment>
`;
